src/elements/card/Card.scss

```diff
@@ -24,20 +24,28 @@ novo-card {
             .title {
                 display: flex;
                 align-items: center;
+                height: 1.75em;
+                overflow: hidden;
 
                 i.bhi-move {
                     color: $light;
                     margin-right: 10px;
+                    height: 1.3em
                     cursor: pointer;
+
                 }
 
                 h3 {
                     padding: 0;
+                    display: -webkit-box;
```

Comment:
Just tried that. The card container doesn't respect it's own size with that implementation. The card grows off of the screen instead of wrapping.